package.json

```diff
@@ -201,6 +202,7 @@
       "allowedVersions": {
         "@typescript-eslint/parser": "^7.0.0",
         "esbuild-sass-plugin>esbuild": "^0.16.7",
+        "eslint": "^8.0.0",
```

Comment:
`@eslint-react/eslint-plugin` expects eslint v9 as a peer dependency, but it works just fine eslint v8.

We can remove this override whenever we upgrade to eslint v9 but I would still wait on that to major version to stabilize.